### DISCUSSION TITLE
Error Code in Sewers when entering red door

### DISCUSSION THREAD

**Discussion by JSmonsterr** [IntelOrca/biorand#680] [General]
Hey there!  I was playing this with a group tonight and hit a road block.  I'm getting the following error code right near the checkpoint door in the sewers:  Couldn't Find File /horr/jpn/24.tmd 

![image](https://github.com/IntelOrca/biorand/assets/152264421/9ddca2b9-a3e4-42ff-9957-25e88ae6f8cd)

Digging through the files, I can't even find the file path to replace the file.  

Here is my seed for the game as well, just in case it's tied to that.  

R710-67UK-XXHH00C5Z9Y0007JZZZZZZZ5ZZZZZZZZZZZZZZZZZZZZZ

Thanks again for your help!

**Comment by IntelOrca** [comment]
Do you get this error in the vanilla game?
Did you download the game from biorand.net?

**Reply by JSmonsterr** [reply]
I actually downloaded it from Biorand.net - I'll test vanilla really quick too.

**Reply by JSmonsterr** [reply]
Vanilla appears to be ok - I pulled the files over to a different computer and confirmed the Red door next to the sewer stairs works.  I'm assuming it's the seed itself.  I'll try running the seed again.

**Reply by IntelOrca** [reply]
Would you be able to post a debug saved game right before the door?

**Reply by JSmonsterr** [reply]
Sure - what are the steps to pull the debug?

**Reply by IntelOrca** [reply]
Enable debug menu if you haven't already done that. Open in game, F5 and save the game then upload the save file.

**Reply by JSmonsterr** [reply]
BTW - We are good!  It's the build of RE1 I had.  I redownloaded and the seed was working fine after.
